webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -3203,7 +3203,7 @@ const AdminDefinition: AdminDefinitionType = {
                 url: 'site_config/wrangler',
                 title: t('admin.sidebar.move_thread'),
                 title_default: 'Move Thread (Beta)',
-                isHidden: it.any(it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.SITE.POSTS)), it.configIsFalse('FeatureFlags', 'MoveThreadsEnabled')),
+                isHidden: it.any(it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.SITE.POSTS)), it.configIsFalse('FeatureFlags', 'MoveThreadsEnabled'), it.not(it.any(it.licensedForSku('enterprise'), it.licensedForSku('e20')))),
```

Comment:
So many double negatives 

If any of the following are true, the menu item is hidden:
- The user DOES NOT have permission
- The feature flag IS FALSE
- If all of the following are NOT TRUE
-- The license is enterprise
-- The license is e20

Reply:
Should also add Professional ([reference](https://docs.google.com/document/d/1UCaQkeahlSa0Uv_TiGB7D3LK1vBCEsfvjkU6cV5xeZE/edit#heading=h.pcx78kd0udng))

Reply:
In that case, I've updated the PR to simply enforce the existence of a license. Pro/Ent/E10/E20 will all be covered by this

Reply:
Sounds good